lib/releaselib_test.sh

```diff
@@ -1,26 +1,54 @@
 #!/usr/bin/env bash
+#
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 #
 # releaselib.sh unit tests
 #
-source $(dirname $(readlink -ne $BASH_SOURCE))/common.sh
-source $TOOL_LIB_PATH/gitlib.sh
-source $TOOL_LIB_PATH/releaselib.sh
+# shellcheck source=./lib/common.sh
+source "$(dirname "$(readlink -ne "${BASH_SOURCE[0]}")")/common.sh"
+# shellcheck source=./lib/testing.sh
+source "${TOOL_LIB_PATH}/testing.sh"
+# shellcheck source=./lib/gitlib.sh
+source "${TOOL_LIB_PATH}/gitlib.sh"
+# shellcheck source=./lib/releaselib.sh
+source "${TOOL_LIB_PATH}/releaselib.sh"
+
+set -e
```

Comment:
Prefer:
```
set -o errexit
set -o nounset
set -o pipefail
```

and set options immediately after file header.

Reply:
https://github.com/kubernetes/release/pull/798#discussion_r307699214